### ISSUE TITLE
[JENKINS-5385] Tag this build failed due to svn commit hook for comment

### ISSUE THREAD

**Issue by jenkins-infra-bot** [jenkinsci/subversion-plugin#653]
<p>When using subversion tagging the Tag this build feature in build history doesn't allow comments to be added for the commit. If SVN commit hook is installed and commit comment is required, tagging will fail:</p>

<pre>
Build #32

Tagging is in progress:

Tagging https:<span class="code-comment">//www.dbcde.com/svn/repos/relaut/branches/uniport-maven (rev.53) to https://www.dbcde.com/svn/repos/relaut/tags/Uniport-32
</span>
Failed to tag

org.tmatesoft.svn.core.SVNException: svn: Commit failed (details follow):

svn: Commit blocked by pre-commit hook (exit code 1) with output:

Associations are required on commit.

svn: MERGE of <span class="code-quote">'/svn/repos/relaut'</span>: 409 Conflict (https:<span class="code-comment">//www.dbcde.com)
</span>	at org.tmatesoft.svn.core.internal.wc.SVNErrorManager.error(SVNErrorManager.java:64)
	at org.tmatesoft.svn.core.internal.wc.SVNErrorManager.error(SVNErrorManager.java:51)
	at rg.tmatesoft.svn.core.internal.wc.SVNCopyDriver.copyReposToRepos(SVNCopyDriver.java:325)
	at org.tmatesoft.svn.core.internal.wc.SVNCopyDriver.setupCopy(SVNCopyDriver.java:563)
	at org.tmatesoft.svn.core.wc.SVNCopyClient.doCopy(SVNCopyClient.java:426)
	at hudson.scm.SubversionTagAction$TagWorkerThread.perform(SubversionTagAction.java:231)
	at hudson.model.TaskThread.run(TaskThread.java:118)

Completed
</pre>

---
<details><summary><i>Originally reported by <a href="https://issues.jenkins.io/secure/ViewProfile.jspa?name=sergey_shikov">sergey_shikov</a>, imported from: <a class="original-jira-link" href="https://issues.jenkins.io/browse/JENKINS-5385" target="_blank">Tag this build failed due to svn commit hook for comment</a></i></summary>
<i><ul>
<li><b>status</b>: Closed
<li><b>priority</b>: Major
<li><b>component(s)</b>: subversion-plugin
<li><b>resolution</b>: Duplicate
<li><b>resolved</b>: 2010-01-26T07:59:15+00:00
<li><b>votes</b>: 0
<li><b>watchers</b>: 2
<li><b>imported</b>: 20251212-090250
</ul></i>
<details><summary>Raw content of original issue</summary>

<pre>
<p>When using subversion tagging the Tag this build feature in build history doesn't allow comments to be added for the commit. If SVN commit hook is installed and commit comment is required, tagging will fail:</p>

<div class="code panel" style="border-width: 1px;"><div class="codeContent panelContent">
<pre class="code-java">Build #32

Tagging is in progress:

Tagging https:<span class="code-comment">//www.dbcde.com/svn/repos/relaut/branches/uniport-maven (rev.53) to https://www.dbcde.com/svn/repos/relaut/tags/Uniport-32
</span>
Failed to tag

org.tmatesoft.svn.core.SVNException: svn: Commit failed (details follow):

svn: Commit blocked by pre-commit hook (exit code 1) with output:

Associations are required on commit.

svn: MERGE of <span class="code-quote">'/svn/repos/relaut'</span>: 409 Conflict (https:<span class="code-comment">//www.dbcde.com)
</span>	at org.tmatesoft.svn.core.internal.wc.SVNErrorManager.error(SVNErrorManager.java:64)
	at org.tmatesoft.svn.core.internal.wc.SVNErrorManager.error(SVNErrorManager.java:51)
	at rg.tmatesoft.svn.core.internal.wc.SVNCopyDriver.copyReposToRepos(SVNCopyDriver.java:325)
	at org.tmatesoft.svn.core.internal.wc.SVNCopyDriver.setupCopy(SVNCopyDriver.java:563)
	at org.tmatesoft.svn.core.wc.SVNCopyClient.doCopy(SVNCopyClient.java:426)
	at hudson.scm.SubversionTagAction$TagWorkerThread.perform(SubversionTagAction.java:231)
	at hudson.model.TaskThread.run(TaskThread.java:118)

Completed
</pre>
</div></div></pre>
</details>
</details>

**Comment by jenkins-infra-bot** [comment]
<details><summary><i>sergey_shikov:</i></summary>

<ul>
<li><i>Original <a class="original-jira-link" href="https://issues.jenkins.io/browse/JENKINS-5385?focusedId=135330&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-135330">comment link</a></i>
<li><details><summary><i>Raw content of original comment:</i></summary>

<pre>
<p>This is <b>not</b> a duplicate of <b><a href="https://issues.jenkins.io/browse/JENKINS-1725" title="Tag this build - allow comments to be added" class="issue-link" data-issue-key="JENKINS-1725"><del>JENKINS-1725</del></a> Tag this build - allow comments to be added</b>.</p>

<p><a href="https://issues.jenkins.io/browse/JENKINS-1725" title="Tag this build - allow comments to be added" class="issue-link" data-issue-key="JENKINS-1725"><del>JENKINS-1725</del></a> requests a nice-to-have feature, but this one is a <b>bug</b> - because tagging is impossible if post-commit hook enabled and commens required. In other words, tagging doesn't work at all in this case.</p>
</pre>
</details>

</ul>
</details>

<p>This is <b>not</b> a duplicate of <b><a href="https://issue-redirect.jenkins.io/issue/1725" title="Tag this build - allow comments to be added" class="issue-link" data-issue-key="<a class="jira-plain-text-key" href="https://issue-redirect.jenkins.io/issue/1725">JENKINS-1725</a>"><del>JENKINS-1725</del></a> Tag this build - allow comments to be added</b>.</p>

<p><a href="https://issue-redirect.jenkins.io/issue/1725" title="Tag this build - allow comments to be added" class="issue-link" data-issue-key="<a class="jira-plain-text-key" href="https://issue-redirect.jenkins.io/issue/1725">JENKINS-1725</a>"><del>JENKINS-1725</del></a> requests a nice-to-have feature, but this one is a <b>bug</b> - because tagging is impossible if post-commit hook enabled and commens required. In other words, tagging doesn't work at all in this case.</p>

**Comment by jenkins-infra-bot** [comment]
<details><summary><i>natalia_leon:</i></summary>

<ul>
<li><i>Original <a class="original-jira-link" href="https://issues.jenkins.io/browse/JENKINS-5385?focusedId=237171&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-237171">comment link</a></i>
<li><details><summary><i>Raw content of original comment:</i></summary>

<pre>
<p>Good day,</p>

<p>As this error is resolved, I tried but the plugin will not let modify the comment.</p>
</pre>
</details>

</ul>
</details>

<p>Good day,</p>

<p>As this error is resolved, I tried but the plugin will not let modify the comment.</p>

**Comment by jenkins-infra-bot** [comment]
<i>[Original `duplicates` from Jira: <a class="jira-relationship" href="https://issue-redirect.jenkins.io/issue/1725">JENKINS-1725</a>]</i>
